index.html

```diff
@@ -3682,7 +3682,51 @@ <h3>Evidence</h3>
   }</span>
 }
         </pre>
-
+ <pre class="example nohighlight" title="Example of evidence supporting a skill achievement credential">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2",
+    "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.3.json"
```

Comment:
can this URL be relied upon for a long time? or should it not matter, since it's an example?

Reply:
It doesn't really matter since it's an example. That said, 1EdTech (formerly known as: IMS Global) is a global standards setting organization for the education sector (effectively, W3C's equivalent in the education space). It maintains that URL and context and it's expected that the URL/context is long-term stable.